main.lua

```diff
@@ -46,40 +46,10 @@ end
 function love.keypressed(k)
 	result = {}
 	i = 1
-
-	if k == "0" then
-		value = value*10 + 0
-	end
-	if k == "1" then
-		value = value*10 + 1
-	end
-	if k == "2" then
-		value = value*10 + 2
-	end
-	if k == "3" then
-		value = value*10 + 3
-	end
-	if k == "4" then
-		value = value*10 + 4
-	end
-	if k == "5" then
-		value = value*10 + 5
-	end
-	if k == "6" then
-		value = value*10 + 6
-	end
-	if k == "7" then
-		value = value*10 + 7
-	end
-	if k == "8" then
-		value = value*10 + 8
-	end
-	if k == "9" then
-		value = value*10 + 9
-	end
-
```

Comment:
... **why**

Reply:
wow, so you're just going to delete it? Those if's are crucial to the way it works, with no replacement you won't be able to input numbers into the program. 

that was the first idea that came into my head on how to solve having a number input, but i have thought of a replacement. rather than having these multiple if's, i can have:
```
local n = (tonumber(k) and tonumber(k)) or 0
value = value*10 + n
```
Setting n to the number that is typed in, and then doing the uniform addition and multiplication required.

I will not be accepting your pull request, as it does nothing but break my project, but i will be committing a replacement for these if's so as to clean up my code.

Reply:
It would have been more respectful to open this as an issue, rather than a destructive pull request

Reply:
Sorry for not seeing your fix to the issue and getting upset at you, but yes, it would have given an error anyway. You still got me to investigate other solutions however so it wasn't all in vein :D